src/generated/dispatch.cpp

```diff
@@ -30,7 +30,11 @@
 namespace crash_diagnostic_layer {
 
 void InitInstanceDispatchTable(VkInstance instance, PFN_vkGetInstanceProcAddr pa, InstanceDispatchTable *dt) {
+#ifdef VK_USE_PLATFORM_ANDROID_KHR
```

Comment:
How about instead making a `global_instance_param` local variable that is either `instance` or `VK_NULL_HANDLE depending on the platform?

Reply:
Updated